sdk/build.gradle

```diff
@@ -8,7 +8,7 @@ apply plugin: "org.jetbrains.dokka"
 apply plugin: 'io.radar.mvnpublish'
 
 ext {
-    radarVersion = "3.2.3"
+    radarVersion = '3.3.0-beta.2'
```

Comment:
Unrelated to these changes, but does this need to match the release that gets cut from github?

Reply:
No. If you look at line 19, `radarVersion` will get changed to the GitHub tag, if present.